pkg/agent/controller/serviceexternalip/controller_test.go

```diff
@@ -357,6 +368,12 @@ func TestCreateService(t *testing.T) {
 			if tt.overrideHashFn != nil {
 				c.fakeMemberlistCluster.hashFn = tt.overrideHashFn
 			}
+			for service, state := range tt.previousExternalIPStates {
```

Comment:
should it set previousExternalIPStates to `c.externalIPStates` to make them consistent? Otherwise it looks confusing why previous state was delete from expected state after syncing another service.

Reply:
updated. thanks!